tests/classes/event_properties/test_event_property_validations.py

```diff
@@ -0,0 +1,64 @@
+from lighthouse.classes.event_properties.interfaces import EventPropertyAbstract  # type: ignore
+from lighthouse.classes.event_properties.validations import SimpleEventPropertyMixin  # type: ignore
+
+
+class DummyEventProperty(EventPropertyAbstract, SimpleEventPropertyMixin):
+    def add_to_warehouse_message(self):
+        return None
+
+    def validate(self):
+        return self._validate
+
+    def value(self):
+        return self._value
+
+
+class TestEventPropertyValidations:
+    def test_validate_param_not_missing(self):
+        test = DummyEventProperty({"asdf": 1})
+        test.validate_param_not_missing("asdf")
+        assert test.validate() is True
+        assert test.errors == []
+        test.validate_param_not_missing("jkl")
+        assert test.validate() is False
+        assert test.errors == ["'jkl' is missing"]
+
+    def test_validate_param_not_empty(self):
```

Comment:
Does this also catch if the param is `None`?

Reply:
No, that is checked in the main class.